Resources/Views/speaker.leaf

```diff
@@ -25,6 +25,18 @@
 
 #embed("videosSection")
 
+#if(count(videos) == 0) {
+<div class="container">
+    <div class="row">
+        <div class="col-sm">
+            <div class="form-container" style="text-align: center">
```

Comment:
Lets use the no-content-container instead of form-container. It is better to have no-content-container

Reply:
Ok, I've added .no-content-container back in and have made it copy the style from .form-container